index.html

```diff
@@ -114,7 +114,32 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-        
+
+                <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Kat M</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
```

Comment:
If you don't have twitter, change the handle to github and use your github user profile instead!
```suggestion
            <i class="fab fa-github"></i>
```